v4/integrations/nrmicro/nrmicro.go

```diff
@@ -199,13 +198,8 @@ func SubscriberWrapper(app *newrelic.Application) server.SubscriberWrapper {
 			return fn
 		}
 		return func(ctx context.Context, m server.Message) (err error) {
-			namer := internal.MessageMetricKey{
-				Library:         "Micro",
-				DestinationType: string(newrelic.MessageTopic),
-				DestinationName: m.Topic(),
-				Consumer:        true,
-			}
-			txn := app.StartTransaction(namer.Name())
+			name := m.Topic() + " receive"
+			txn := app.StartTransaction(name)
```

Comment:
This change updates how we name message consumer transactions. This matches how we name the producer spans.

Reply:
See the OpenTelemetry [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/messaging.md) for messaging systems doc for more info!